tests/testflows/rbac/requirements/requirements.py

```diff
@@ -0,0 +1,6132 @@
+# These requirements were auto generated
+# from software requirements specification (SRS)
+# document by TestFlows v1.6.200629.1155222.
+# Do not edit by hand but re-generate instead
+# using 'tfs requirements generate' command.
```

Comment:
It would be nice if you make your generation tool write here the path to the `requirements.md` file.

Reply:
Noted. But not sure that if it would be that useful as it would just be a path to somewhere on my machine. I think we if add the source for the requirements to this folder it will be self explanatory.